ui/src/main/resources/edu/wpi/grip/ui/templates/python/operations/Filter_Contours.vm

```diff
@@ -17,9 +17,9 @@
             solid = 100 * area / cv2.contourArea(hull)
             if (solid < solidity[0] or solid > solidity[1]):
                 continue
-            if (contour.size() < minVertexCount or contour.size() > maxVertexCount):
+            if (len(contour) < minVertexCount or len(contour) > maxVertexCount):
                 continue
-            ratio = bb.width / bb.height
+            ratio = (float)(w) / (float)(h)
```

Comment:
Why cast to `float` here when you cast to `double` in `FilterContours`?. You also only need to cast one of these